Dockerfile

```diff
@@ -4,36 +4,36 @@ ENV CGO_ENABLED=0
 ENV GOOS=linux
 RUN useradd -u 10001 connect
 
-WORKDIR /go/src/github.com/redpanda-data/connect-plugin-example/
+WORKDIR /go/src/github.com/redpanda-data/redpanda-connect-plugin-example/
 # Update dependencies: On unchanged dependencies, cached layer will be reused
-COPY go.* /go/src/github.com/redpanda-data/connect-plugin-example/
+COPY go.* /go/src/github.com/redpanda-data/redpanda-connect-plugin-example/
 RUN go mod download
 
 # Build
-COPY . /go/src/github.com/redpanda-data/connect-plugin-example/
+COPY . /go/src/github.com/redpanda-data/redpanda-connect-plugin-example/
 
 # Tag timetzdata required for busybox base image:
 # https://github.com/benthosdev/benthos/issues/897
```

Comment:
Shouldn't issue links now go to https://github.com/redpanda-data/connect/issues?